source/common/runtime/runtime_features.cc

```diff
@@ -179,7 +180,6 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_use_network_type_socket_option);
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_quic_disable_client_early_data);
 
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_ext_proc_graceful_grpc_close);
-
```

Comment:
nit: add empty line back to keep the next block visually separated.

Reply:
done